ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/capturePoint/LinearMomentumRateControlModule.java

```diff
@@ -292,6 +292,7 @@ public LinearMomentumRateControlModule(CenterOfMassStateProvider centerOfMassSta
    public void reset()
    {
       desiredLinearMomentumRateWeight.set(linearMomentumRateWeight);
+      initializeOnStateChange = true;
```

Comment:
This helps fix a bug where the controller fails to balance because of trying to apply a rate limit to the CMP after falling.